dandiapi/api/views/users.py

```diff
@@ -67,12 +67,12 @@ def social_account_to_dict(social_account: SocialAccount):
 def serialize_user(user: User):
     """Serialize a user that's been annotated with a `social_account_data` field."""
     username = user.username
-    name = f'{user.first_name} {user.last_name}'.strip()
+    name = user.get_full_name()
```

Comment:
This [function](https://github.com/django/django/blob/c496a7df64ec0a80c2bab7f155aad567155fb37b/django/contrib/auth/models.py#L501-L506) on the `User` model is exactly the same as what we're doing here.